core/src/main/scala/org/apache/spark/metrics/sink/opentelemetry/OpenTelemetryPushReporter.scala

```diff
@@ -0,0 +1,401 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.metrics.sink.opentelemetry
+
+import java.nio.file.{Files, Paths}
+import java.util.{Locale, SortedMap}
+import java.util.concurrent.TimeUnit
+
+import com.codahale.metrics._
+import io.opentelemetry.api.common.Attributes
+import io.opentelemetry.api.metrics.{DoubleGauge, DoubleHistogram, LongCounter}
+import io.opentelemetry.api.trace.propagation.W3CTraceContextPropagator
+import io.opentelemetry.context.propagation.ContextPropagators
+import io.opentelemetry.exporter.otlp.metrics.OtlpGrpcMetricExporter
+import io.opentelemetry.sdk.OpenTelemetrySdk
+import io.opentelemetry.sdk.metrics.SdkMeterProvider
+import io.opentelemetry.sdk.metrics.export.PeriodicMetricReader
+import io.opentelemetry.sdk.resources.Resource
+
+
+private[spark] class OpenTelemetryPushReporter (
+    registry: MetricRegistry,
+    pollInterval: Int = 10,
+    pollUnit: TimeUnit = TimeUnit.SECONDS,
+    host: String = "http://localhost",
+    port: String = "4317",
+    headersMap: Map[String, String] = Map(),
+    attributesMap: Map[String, String] = Map(),
+    trustedCertificatesPath: String,
+    privateKeyPemPath: String,
+    certificatePemPath: String
+) extends ScheduledReporter (
```

Comment:
Please move the starting `)` to the end of the above line.
- https://github.com/databricks/scala-style-guide